src/main/java/UI.java

```diff
@@ -0,0 +1,128 @@
+                "____________________________________________________________\n";
+        System.out.println(bye);
+    }
+
+    public void printIDK() {
+        System.out.println("____________________________________________________________\n" +
+                "     OOPS!!! I'm sorry, but I don't know what that means :(\n" +
+                "____________________________________________________________");
+    }
+
+    public void printLine() {
+        System.out.println("____________________________________________________________\n");
+    }
+
+    public void printList(TaskList taskList) {
+        System.out.println("____________________________________________________________\n" +
+                "Here are the tasks in your list:");
+        int count = 1;
+        for (int i = 0; i < taskList.size(); i++) {
+            Task t = taskList.get(i);
+            System.out.println(count + "." + t.toString());
+            count++;
+        }
+        System.out.println("____________________________________________________________\n");
+    }
+
+    public void printDone(Task task) {
+        System.out.println("____________________________________________________________\n" +
+                "Congratulations! I've marked this task as done: ");
+        System.out.println("[" + task.getStatusIcon() + "] " + task.getDescription());
+        System.out.println("____________________________________________________________\n");
+    }
+
+    public void printAddTask(Task task, int taskListSize) {
+        System.out.println("____________________________________________________________\n" +
+                "Got it. I've added this task: ");
+        System.out.println(task.toString());
+        System.out.println("Now you have " +  taskListSize + " task(s) in the list.");
+        System.out.println("____________________________________________________________\n");
+    }
+
+    public void printDelete(Task deletedTask, int taskListSize) {
+        System.out.println("____________________________________________________________\n" +
+                "Noted. I've removed this task: \n" +
+                deletedTask.toString() + "\n" +
+                "Now you have " + (taskListSize - 1) + " task(s) in the list.\n" +
+                "____________________________________________________________");
+    }
+
+    public void printFind(TaskList findList) {
+        System.out.println("____________________________________________________________\n" +
+                "Here are the matching tasks in your list:");
+        int count = 1;
+        for (int i = 0; i < findList.size(); i++) {
+            Task t = findList.get(i);
+            System.out.println(count + "." + t.toString());
+            count++;
+        }
+        System.out.println("____________________________________________________________\n");
+    }
+
+    public void throwInputError(String taskType) throws DukeException {
```

Comment:
Perhaps you could store pretty printing elements (e.g. the line segmentation) as a constant in the class?